File: samples/java_layout/src/main/java/io/customer/android/sample/java_layout/utils/Logger.java
package io.customer.android.sample.java_layout.utils;

import android.util.Log;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

public class Logger {
    private static final String TAG = "JAVA_LAYOUT";

    public void d(@NonNull String message) {
        Log.d(TAG, message);
    }

    public void v(@NonNull String message) {
        this.v(message, null);
    }

    public void v(@NonNull String message, @Nullable Throwable throwable) {
        Log.v(TAG, message, throwable);
    }

    public void e(@NonNull String message) {
        this.e(message, null);
    }

    public void e(@NonNull String message, @Nullable Throwable throwable) {
        Log.e(TAG, message, throwable);
    }
}

File: samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/util/Logger.kt
package io.customer.android.sample.kotlin_compose.util

import android.util.Log

class Logger {
    companion object {
        private const val TAG = "KOTLIN_COMPOSE"
    }

    fun d(message: String) {
        Log.d(TAG, message)
    }

    fun v(message: String) {
        v(message, null)
    }

    fun v(message: String, throwable: Throwable?) {
        Log.v(TAG, message, throwable)
    }

    fun e(message: String) {
        e(message, null)
    }

    fun e(message: String, throwable: Throwable?) {
        Log.e(TAG, message, throwable)
    }
}
